app/commands/decidim/participatory_documents/admin/final_sections_publish.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryDocuments
+    module Admin
+      class FinalSectionsPublish < Rectify::Command
+        # Public: Initializes the command.
+        #
+        # document - The document to publish.
+        def initialize(document)
+          @document = document
+        end
+
+        # Executes the command. Broadcasts these events:
+        #
+        # - :ok when everything is valid.
+        # Returns nothing.
+        def call
+          final_publish
+
+          broadcast(:ok)
+        end
+
+        private
+
+        attr_reader :document
+
+        def final_publish
+          document.update!(final_publish: true)
+          document.save!
+        end
```

Comment:
Instead of doing this, is better to use the `Decidim::Publicable` concern in the model. This gives you methods like this:
```ruby
document.publish!
```